djstripe/models.py

```diff
@@ -456,16 +442,13 @@ def sync_charges(self, cu=None, **kwargs):
     def sync_current_subscription(self, cu=None):
         cu = cu or self.stripe_customer
         sub = cu.subscription
+        plan_instance = Plan.objects.get(stripe_id=sub.plan.id)
```

Comment:
Move this to line 447 and add a blank line after it. If there's no sub, there's no plan.